app/client/components/pages/creditTransferListPage.jsx

```diff
@@ -29,7 +29,7 @@ const mapDispatchToProps = (dispatch) => {
 
 class CreditTransferListPage extends React.Component {
   componentDidMount() {
-      this.props.loadCreditTransferList({per_page:50, page:1})
+      this.props.loadCreditTransferList()
```

Comment:
I thought we were loading in the first 200?

Reply:
yeah 200 or 500 is good, will update